<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="portaSerialLbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Porta Serial</string>
    </property>
   </widget>
   <widget class="QLabel" name="acoesLbl">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ações</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLbl">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Comunicação Serial</string>
    </property>
   </widget>
   <widget class="QLabel" name="portaSerialLbl_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comandos</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>531</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="comandoLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="enviarDadoBtn">
       <property name="text">
        <string>Enviar Dado</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="comandoTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>553</width>
      <height>180</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="portaSerialLbl_5">
           <property name="text">
            <string>Portas:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="portaSerialCombBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="baudrateLbl">
           <property name="text">
            <string>Baudrate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="baudrateComboBox">
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="conectarBtn">
         <property name="text">
          <string>Conectar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>switch.png</normaloff>switch.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="shortcut">
          <string>F</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="desconectarBtn">
         <property name="text">
          <string>Desconectar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>switch-2.png</normaloff>
           <normalon>switch-2.png</normalon>switch-2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="ajudaBtn">
       <property name="text">
        <string>Ajuda</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>question-mark-on-a-circular-black-background.png</normaloff>question-mark-on-a-circular-black-background.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sairBtn">
       <property name="text">
        <string>Sair</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>logout.png</normaloff>logout.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="versionBtn">
         <property name="text">
          <string>Versão</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hardwareBtn">
         <property name="text">
          <string>Hardware</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ligaLedBtn">
         <property name="text">
          <string>Liga LED</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>led.png</normaloff>led.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="desligaLedBtn">
         <property name="text">
          <string>Desliga LED</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>led-diode.png</normaloff>led-diode.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="analogicaBtn">
         <property name="text">
          <string>Ler Entrada Analógica</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
